resources/css/error.css

```
@@ -0,0 +1,1893 @@
+
+        .dark\:bg-zinc-800 {
+            background-color: var(--color-zinc-800)
+        }
+
+        .dark\:bg-zinc-900 {
+            background-color: var(--color-zinc-900)
+        }
+
+        .dark\:bg-linear-to-b {
+            --tw-gradient-position: to bottom in oklab;
+            background-image: linear-gradient(var(--tw-gradient-stops))
```

Biome (1.9.4)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)